src/zeroD/ReactorFactory.cpp

```diff
@@ -68,7 +68,8 @@ void ReactorFactory::deleteFactory() {
 
 shared_ptr<ReactorBase> newReactor(const string& model)
 {
-    // deprecation warning is handled in Python API
+    warn_deprecated("newReactor",
+        "Starting in Cantera 3.1, the reactor contents must not be empty.");
```

Comment:
```suggestion
        "Starting in Cantera 3.1, the reactor contents must be provided in the constructor.");
```

Reply:
I think the original message introduced in #1663 was not ideal; non-empty reactor contents are being deprecated in 3.1, and removed thereafter.